e2e/stack_test.go

```diff
@@ -455,6 +369,155 @@ func erc20RollupFlow(t *testing.T, stack *e2e.StackConfig) {
 	requireERC20IsMinted(t, stack, userAddr, weth9Address.String(), hexutil.EncodeBig(wethL2Amount))
 
 	t.Log("Monomer can ingest ERC-20 deposit txs from L1 and mint ERC-20 tokens on L2")
+
+	////////////////////////////////
+	////// ERC-20 WITHDRAWALS //////
+	////////////////////////////////
+
+	// deposit ETH to the user's account on L2 to pay for gas
+	depositAmount := big.NewInt(params.Ether)
+	depositTx, err := PadGasEstimate(
+		createL1TransactOpts(t, stack, userPrivKey, l1signer, depositAmount),
+		1.1,
```

Comment:
Remind me what this value does and how it is selected?

Reply:
1.1 is the padding factor used to pad the gas estimate to prevent the test from failing due to out-of-gas errors. This is based on a similar function used in the [op-e2e package](https://github.com/ethereum-optimism/optimism/blob/24a8d3e06e61c7a8938dfb7a591345a437036381/op-e2e/tx_helper.go#L38).